core/src/com/unciv/ui/newgamescreen/GameOptionsTable.kt

```diff
@@ -64,14 +66,16 @@ class GameOptionsTable(
         }).row()
         addVictoryTypeCheckboxes()
 
+        val religionDisabledByRuleset = (ruleset.eras[gameParameters.startingEra]!!.hasUnique(UniqueType.DisablesReligion)
+                || ruleset.modOptions.uniques.contains(ModOptionsConstants.disableReligion))
```

Comment:
Move this into the addReligionCheckbox, since it's not being used by anything outside of it

Reply:
Also, do we update gameOptionsTable when selecting a starting era? Because if we don't then the first part will be locked onto the default starting era, and won't change with it.